### PR TITLE
[F-1] ats-faisal

### PR DESCRIPTION
**Pull request by FAISAL-K-ALOTAIBI**
new code

### CODE REVIEW

**Review by XMjeedX — APPROVED** [review]

**Review by ZDZD-S — APPROVED** [review]

File: ATS(faisal)/ats.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import csv
+import os
+
+# Define the path to the CSV file
+CSV_FILE = 'jobs.csv'
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:5]: This should be input from user, or add it to "utils/constants.py" file

File: ATS(faisal)/ats.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+CSV_FILE1 = 'candidate.csv'
+
+# Create CSV file if it doesn't exist
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:8]: We need class for file managment

File: ATS(faisal)/ats.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+# Create CSV file1 if it doesn't exist
+if not os.path.exists(CSV_FILE1):
+    with open(CSV_FILE1, 'w', newline='') as csvfile1:
+        fieldnames = ['candidate: name', 'email', 'phone_number']
+        writer = csv.DictWriter(csvfile1, fieldnames=fieldnames)
+        writer.writeheader()
+
+# function to candidate list 
+def list_candidate():
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:23]: Need class for candidates functions

File: ATS(faisal)/ats.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+def add_candidate(full_name, email, phone_number):
+    with open(CSV_FILE1, 'a',newline='' ) as csvfile1:
+        fieldnames = ['full_name', 'email', 'phone_number']
+        writer = csv.DictWriter(csvfile1, fieldnames=fieldnames)
+        writer.writerow({'full_name': full_name, 'email': email, 'phone_number': phone_number})
+    print(" candidate added successfully ")
+
+
+# Function to list jobs
+def list_jobs():
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:41]: Need class for jobs functions

File: ATS(faisal)/ats.py [hunk 5]
```diff
@@ -0,0 +1,86 @@
+import csv
+import os
+
+# Define the path to the CSV file
+CSV_FILE = 'jobs.csv'
+CSV_FILE1 = 'candidate.csv'
+
+# Create CSV file if it doesn't exist
+if not os.path.exists(CSV_FILE):
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:9]: no need to check, you can add them to "csv/jobs.csv"

File: ATS(faisal)/ats.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+# Main function
+def main():
+    while True:
+        print("\n1. List Jobs")
+        print("2. Add Job")
+        print("3. list candidate")
+        print("4. Add candidate")
+        print("5. Exit")
+        choice = input("Enter your choice: ")
+        if choice == '1':
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:66]: use python Enum, instead of direct numbers "1"

File: ATS(faisal)/jobs.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+title,description
+suply performance ,respancpol for the suply .
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/jobs.csv:2]: This file should be empty

**Review by MohammadModallalBloovo — CHANGES_REQUESTED** [review]

File: ATS(faisal)/ats.py [hunk 8]
```diff
@@ -0,0 +1,86 @@
+import csv
```
Comment (MohammadModallalBloovo) [inline ATS(faisal)/ats.py:1]: requirements.txt
.gitignore
errors.py
constants.py
slack.py

Comment (MohammadModallalBloovo) [reply ATS(faisal)/ats.py:1]: slack webhook

**Pull request closed without merging**